RevenueCatUI/Data/TemplateViewConfiguration.swift

```diff
@@ -236,54 +236,56 @@ extension TemplateViewConfiguration.PackageConfiguration {
-
-                    let tierName = localization.tierName ?? ""
-
-                    let filteredPackages = Self.processPackages(
-                        from: packages,
-                        filter: tier.packages,
-                        activelySubscribedProductIdentifiers: activelySubscribedProductIdentifiers,
-                        localization: localization,
-                        locale: locale,
-                        showZeroDecimalPlacePrices: showZeroDecimalPlacePrices
-                    )
+            let filteredTiers: [(PaywallData.Tier, (MultiPackage, String))] = try tiers.compactMap { tier in
+                guard let localization = localization[tier.id] else {
+                    throw TemplateError.missingLocalization(tier)
+                }
 
-                    guard let firstPackage = filteredPackages.first else {
-                        Logger.error(Strings.tier_has_no_available_products_for_paywall(tierName))
-                        return nil
-                    }
-                    let defaultPackage = filteredPackages
-                        .first { $0.content.identifier == tier.defaultPackage }
-                    ?? firstPackage
+                let tierName = localization.tierName ?? ""
 
-                    return (
-                        tier,
-                        (
-                            MultiPackage(
-                                first: firstPackage,
-                                default: defaultPackage,
-                                all: filteredPackages
-                            ),
-                            tierName
-                        )
+                let filteredPackages = Self.processPackages(
+                    from: packages,
+                    filter: tier.packages,
+                    activelySubscribedProductIdentifiers: activelySubscribedProductIdentifiers,
+                    localization: localization,
+                    locale: locale,
+                    showZeroDecimalPlacePrices: showZeroDecimalPlacePrices
+                )
+
+                guard let firstPackage = filteredPackages.first else {
+                    Logger.error(Strings.tier_has_no_available_products_for_paywall(tierName))
+                    return nil
+                }
+                let defaultPackage = filteredPackages
+                    .first { $0.content.identifier == tier.defaultPackage }
+                ?? firstPackage
+
+                return (
+                    tier,
+                    (
+                        MultiPackage(
+                            first: firstPackage,
+                            default: defaultPackage,
+                            all: filteredPackages
+                        ),
+                        tierName
                     )
-                },
+                )
+            }
+
+            let filteredTiersMap: [PaywallData.Tier: (package: MultiPackage, tierName: String)] = Dictionary(
```

Comment:
Referring to a dictionary as a "map" isn't idiomatic swift.... our style guide would [recommend](https://github.com/RevenueCat/purchases-ios/blob/0d112d084e13faea89160577dfa086124a7504aa/Contributing/SwiftStyleGuide.swift#L66-L67) something like `packageAndTierNameByTier`. Or you could use `filteredTiersDict` or `filteredTiersLookup`...

Reply:
Ah right, thanks for pointing me to the style guide. Done in https://github.com/RevenueCat/purchases-ios/pull/4243/commits/24c1f689bcf55970b0381420bef0e831ea80a156.